modules/manage/partials/monitor-connectors.adoc

```diff
@@ -41,6 +41,30 @@ The most important metrics to be monitored by alerts are:
 * connector failed tasks
 * connector lag / connector lag rate
 
+== View connector logs
+
+Connector logs are written to the system topic `__redpanda.connectors_logs`. In {ui}, you can view them or pull them with `rpk`. For example:
+
+[,bash]
+----
+# Last 100 messages (most recent)
+rpk topic consume __redpanda.connectors_logs -o -100 -n 100
+
+# Last 10 minutes
+rpk topic consume __redpanda.connectors_logs -o @-10m:end
+
+# Stream new logs only (like tail -f)
+rpk topic consume __redpanda.connectors_logs -o end
+
+# Quick filter by connector name (JSON logs)
+rpk topic consume __redpanda.connectors_logs -o @-10m:end -O json \
+  | jq -r 'select(.message | test("<connector-name>"; "i"))'
```

Comment:
Does `connector-name` have to match a name in the docs, or the Cloud UI, or some other reference? I'm not clear on whether it should be something like `MongoDB` or if `mongodb` is fine, and whether source or sink matters.